src/CoreWCF.AzureQueueStorage/src/CoreWCF/Channels/AzureQueueStorageQueueTransport.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Storage.Queues;
+using Azure.Storage.Queues.Models;
+using CoreWCF.Configuration;
+using CoreWCF.Queue;
+using CoreWCF.Queue.Common;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace CoreWCF.Channels
+{
+    public class AzureQueueStorageQueueTransport : IQueueTransport
+    {
```

Comment:
This should be internal as it's an internal implementation detail. It's generally better to keep things internal unless there's a general use case that an end user developer would need to access it.

Reply:
Yes That makes sense.